tests/test_room.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from classes.room import Room
+from classes.dojo import Dojo
+
+
+class RoomClassTest(unittest.TestCase):
+    def test_add_occupant(self):
+        my_class_instance = Room("Blue", "office", 6)
+        my_dojo_instance = Dojo()
```

Comment:
Have the creation of `my_dojo_instance` be in the class `setUp` because it is repetitive.